test/asynchronous/unified_format.py

```diff
@@ -1443,11 +1440,12 @@ async def _run_scenario(self, spec, uri=None):
         await self.entity_map.create_entities_from_spec(
             self.TEST_SPEC.get("createEntities", []), uri=uri
         )
+        self._cluster_time = None
         # process initialData
         if "initialData" in self.TEST_SPEC:
             await self.insert_initial_data(self.TEST_SPEC["initialData"])
-            self._cluster_time = (await self.client.admin.command("ping")).get("$clusterTime")
-            await self.entity_map.advance_cluster_times()
+            self._cluster_time = self.client._topology.max_cluster_time()
+            await self.entity_map.advance_cluster_times(self._cluster_time)
```

Comment:
I refactored this to remove the "ping" command because it was causing problems with the failpoint for this new test. We can get the clusterTime via `self.client._topology.max_cluster_time()` instead.